addons/mail/static/src/core/common/thread_service.js

```diff
@@ -953,13 +953,9 @@ export class ThreadService {
      * @param {Message} message
      */
     notifyMessageToUser(thread, message) {
-        if (
-            thread.type === "channel" &&
-            message.recipients?.includes(this.store.user) &&
-            message.notIn(thread.needactionMessages)
-        ) {
-            thread.needactionMessages.add(message);
-            thread.message_needaction_counter++;
```

Comment:
you removed the needaction counter stuff?

Reply:
It is already covered by earlier managing of `new_message`, so the counter was increased twice

Reply:
https://github.com/odoo/odoo/blob/17.0/addons/mail/static/src/discuss/core/common/discuss_core_common_service.js#L274